lib/ui/geometry.dart

```diff
@@ -329,43 +327,43 @@ class Offset extends OffsetBase {
 
   /// Compares two Offsets for equality.
   @override
-  bool operator ==(dynamic other) {
+  bool/*!*/ operator ==(dynamic other) {
     return other is Offset
         && other.dx == dx
         && other.dy == dy;
   }
 
   @override
-  int get hashCode => hashValues(dx, dy);
+  int/*!*/ get hashCode => hashValues(dx, dy);
 
   @override
-  String toString() => 'Offset(${dx?.toStringAsFixed(1)}, ${dy?.toStringAsFixed(1)})';
+  String/*!*/ toString() => 'Offset(${dx?.toStringAsFixed(1)}, ${dy?.toStringAsFixed(1)})';
 }
 
 /// Holds a 2D floating-point size.
 ///
 /// You can think of this as an [Offset] from the origin.
 class Size extends OffsetBase {
   /// Creates a [Size] with the given [width] and [height].
-  const Size(double width, double height) : super(width, height);
+  const Size(double/*!*/ width, double/*!*/ height) : super(width, height);
```

Comment:
Again optionally add the asserts.

Reply:
Same here: see the implementation of `OffsetBase`.